kokoro/builds/build.sh

```diff
@@ -205,6 +205,8 @@ if [ -n "$1" ]; then
     cp -v bin/OrbitClientGgp Orbit/collector/
     cp -v lib/libOrbitVulkanLayer.so Orbit/collector/
     cp -v lib/VkLayer_Orbit_implicit.json Orbit/collector/
+    cp -v lib/liborbit.so Orbit/collector/
```

Comment:
The .deb above contains these already. Can you use these? OrbitService is unpacked from there anyway, right?

Reply:
No, the .deb or the OrbitService binary are not used for these tests (nor for `LinuxTracingIntegrationTests`). See http://cs/piper///depot/google3/chrome/cloudcast/orbit/testing/yhiti/tests/BUILD;l=19

It's used in the automated performance testing, but that has a completely different structure.

Reply:
Oh, right, I remember.

But can't you just add 

"dpkg -x /mnt/tmp/Orbit/collector/OrbitProfiler-*.deb /mnt/tmp/",
"cp /mnt/tmp/opt/developer/tools/liborbit.so /mnt/tmp/Orbit/collector/",
"cp /mnt/tmp/opt/developer/tools/liborbituserspaceinstrumentation.so /mnt/tmp/Orbit/collector/",

(after the unzip, between line 18 and 19 in http://cs/piper///depot/google3/chrome/cloudcast/orbit/testing/yhiti/tests/BUILD)

Otherwise we would add ~12mb to the installer - not the end of the world but it seems redundant.

Reply:
Oh yes, of course, I wasn't very well aware of the structure of the .deb. That works well. Thanks!

Reply:
Awesome. :-)